File: CPP/Assignment4/Program1.cpp
/////////////////////////////////////////////////////////////////////////
//
// File name :   Program1.cpp
// Description : Write recursive program which display below pattern.
// Author :      Pallavi Omprakash Malewar
// Date :        27/07/2026
//
/////////////////////////////////////////////////////////////////////////

/*

    Output :    *   *   *   *   *

*/


#include<iostream>
using namespace std;

void Display()
{
    int i = 0;
    for(i = 0 ; i < 5 ; i++)
    {
        cout << "*\t";
    }
    cout<<endl;
}

int main()
{
    Display();
    return 0;
}

File: JAVA/Assignment 5/Program1.java
/////////////////////////////////////////////////////////////////////////////////////////
//                                                                                     //
// Problem Statement : Write Java program which accepts no. of rows and columns from   //
//                     user & display below pattern                                    //
// Author : Pallavi Omprakash Malewar                                                  //
// Date : 30 Jun 2026                                                                  //
//                                                                                     //
/////////////////////////////////////////////////////////////////////////////////////////


/*

    Input  : Hello
    Output : H   e   l   l   o
             H   e   l   l   o
             H   e   l   l   o
             H   e   l   l   o

*/


import java.util.Scanner;

class Pattern
{
    public void Pattern(String s)
    {
        int n = s.length();
        int i = 0, j = 0;

        for(i = 0 ; i < 4 ; i++)
        {
            for(j = 0 ; j < n ; j++)
            {
                System.out.print(s.charAt(j) + "\t");
            }
            System.out.println();
        }
    }
}

class Program1
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.print("Enter the String : ");
        String s = sobj.nextLine();

        Pattern pobj = new Pattern();

        pobj.Pattern(s);
    }
}
